graphene_pydantic/inputobjecttype.py

```diff
@@ -127,11 +137,11 @@ def resolve_placeholders(cls):
         meta = cls._meta
         fields_to_update = {}
         for name, field in meta.fields.items():
-            target_type = field._type
-            if hasattr(target_type, "_of_type"):
-                target_type = target_type._of_type
+            target_type = field.type
+            while hasattr(target_type, "of_type"):
```

Comment:
Curious about why this was switched from `if` to `while`?

Reply:
There were some cases were there was this recursion needed to get the actual type